django/db/backends/sqlite3/creation.py

```diff
@@ -4,11 +4,15 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends.base.creation import BaseDatabaseCreation
+from django.utils.encoding import force_text
 from django.utils.six.moves import input
 
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    def is_database_in_memory(self, database_name):
```

Comment:
I don't know of any nuances about staticmethod and inheritance. If anyone's subclassing the SQLite backend, I've never heard about it.